### PR TITLE
2 connect app to api

### PR DESCRIPTION
**Pull request by OM33**
Hi Good morning, I finished all requirements, please check it out to merge.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -6,6 +6,7 @@ import Typography from "@mui/material/Typography";
 import Paper from "@mui/material/Paper";
 import AppBar from "@mui/material/AppBar";
 import Toolbar from "@mui/material/Toolbar";
+import { useState, useEffect } from "react";
```
Comment (Danswar) [inline src/App.js]: Why are you importing this if you are not going to use it?

**Review by Danswar — APPROVED** [review]
Well done! You were able to change you previous implementation to accommodate to the new requirements!

We can improve this but overall it is ready to go

File: src/App.js (hunk 1 above)
Comment (OM33) [reply src/App.js]: You´re right, thank you for letting me know, I will eliminate this and other libraries I´m not using if that´s the case

**Comment by OM33** [comment]
Hi! I already made all changes, Do I proceed to merge?

**Comment by Danswar** [comment]
> Hi! I already made all changes, Do I proceed to merge?

of course! :)

**Pull request merged**